tools/dump/switches.cpp

```diff
@@ -123,9 +123,14 @@ namespace {
                                  desc{"Emit address values as an explicit segment/offset object"},
                                  cat (how_cat)};
 
+#ifdef PSTORE_IS_INSIDE_LLVM
     opt<std::string> triple{"triple",
                             desc{"The target triple to use for disassembly if one is not known"},
                             init ("x86_64-pc-linux-gnu-repo"), cat (how_cat)};
+    opt<bool> no_disassembly{"no-disassembly",
+                             desc{"Emit executable sections as binary rather than disassembly"},
+                             cat (how_cat)};
```

Comment:
The `no_disassembly` switch option is defined only if `PSTORE_IS_INSIDE_LLVM` is defined. However, the `no_disassembly` is always defined in the parameters structure (line 43 and line 58 in the include/pstore/dump/parameter.hpp file). Is there any reason for that?

Reply:
> Is there any reason for that?

No not really. It isn’t used in that configuration so I guess it could be removed.

Reply:
> No not really. It isn’t used in that configuration so I guess it could be removed.

Commit 371e50f.